React-frontend/src/components/LoginForm.js

```diff
@@ -54,13 +56,13 @@ const useStyles = makeStyles((theme) => ({
         backgroundColor: theme.palette.primary.extraLight,
         margin: theme.spacing(1.5, 0),
         '&:hover': {
-          backgroundColor: theme.palette.primary.light
+            backgroundColor: theme.palette.primary.light
         }
```

Comment:
Format the code properly, Avoid extra space or line!

Reply:
Done.